edb/server/server.py

```diff
@@ -934,18 +944,20 @@ async def _introspect_dbs(self):
                 # information about them.
                 g.create_task(self._early_introspect_db(dbname))
 
-    async def get_patch_count(self, conn):
+    async def get_patch_count(self, conn: pgcon.PGConnection) -> int:
         """Get the number of applied patches."""
-        num_patches = await conn.sql_fetch_val(
+        num_patches_b = await conn.sql_fetch_val(
```

Comment:
I assume this change is to avoid reassigning `num_patches` with a different type, first `str` and then `int`? If so is there an advantage to the intermediate `num_patches` variable rather than returning the expression?

Reply:
Yeah, that's right.

I suspect that the reason it is assigned to an intermediate variable instead of returning it directly is because at some point while I was developing it, something was going wrong and I wanted it split up so I could stick a print or a breakpoint in between, and then I never merged them.

Reply:
Awesome, yeah I have a similar habit especially when developing. Do we prefer this to enable easy debugging or is there no official preference for assignment before returning?

Reply:
No official preference.

I think that constructing stuff so that it is easy to debug on the *macro* scale, but on micro stuff like this, it's easy enough to make the changes on the fly, and code gets read more often than it gets instrumented. I think it's pretty marginal either way. I'm not sure if I really have a personal preference on which version ought to get committed.